packages/user-management-widget/src/lib/widget/mixins/initMixin/initComponentsMixins/initUsersTableMixin.ts

```diff
@@ -30,6 +66,7 @@ export const initUsersTableMixin = createSingletonMixin(
             e.detail.value.map(({ loginIds }) => loginIds),
           );
         });
+        this.#setCustomRenderer();
```

Comment:
Do we still need this?
![image](https://github.com/descope/descope-js/assets/5429033/3ca9eaff-b425-4c41-bb14-657ce4bf44a8)

Reply:
Yes, we do